### DISCUSSION TITLE
How to Load saved MXL files

### DISCUSSION THREAD

**Discussion by samjia533332** [Audiveris/audiveris#660] [Q&A]
Hi 

I saved my book under qilixiang.mxl  
But when I try to open it with "Open Book", I cannot open it. Do you know how to solve this? 

<img width="903" alt="截屏2023-07-21 下午10 25 21" src="https://github.com/Audiveris/audiveris/assets/8285773/f73f93b7-5741-443c-8db1-1a269f17972f">
<img width="337" alt="截屏2023-07-21 下午10 25 38" src="https://github.com/Audiveris/audiveris/assets/8285773/0cbe8f21-9c8e-4923-8563-68bb7d90404c">
<img width="241" alt="截屏2023-07-21 下午10 29 00" src="https://github.com/Audiveris/audiveris/assets/8285773/0b50d782-0ef4-44ac-b45e-1f8f4113e6ee">

**Comment by hbitteur** [comment]
An Audiveris project is saved/reloaded into/from a `.omr` file.
A `.mxl` file is just an Audiveris output using MusicXML format.

Such MusicXML data can be read by external music sequencers such as MuseScore.

There is much more information in `.omr` file than in `.mxl` file.
